src/components/avatar/Avatar.module.scss

```
@@ -0,0 +1,23 @@
+@import '../../utils/variables';
+
+.avatarContainer {
+    width: 60px;
+    height: 60px;
+    background: transparent;
+
+    & > img {
+        width: 100%;
+        height: 100%;
+        border-radius: 50%;
+    }
+    & > i {
+        font-size: 60px;
+    }
+}
+
+@media(max-width: $small) {
+    .avatarContainer {
+        width: 24px;
+        height: 24px;
+    }
+}
```

Oh one style thing!

I think we've been going for nesting media queries as well! So like this:

```suggestion
.avatarContainer {
    width: 60px;
    height: 60px;
    background: transparent;

    & > img {
        width: 100%;
        height: 100%;
        border-radius: 50%;
    }
    & > i {
        font-size: 60px;
    }
    
    @media(max-width: $small) {
        width: 24px;
        height: 24px;
    }
}
```

This just keeps all the styling for `.avatarContainer` in the same place but with the tradeoff of having to write more of these `@media(max-width: $small)`